07LessonSeven/homework/css/style.css

```diff
@@ -14,7 +14,7 @@ main nav {
 }
 span {
     color: white;
-    padding: 10px;
+    padding-right: 15px;
 }
 div.content {
     margin-left: 10%;
```

Comment:
Fantastic! Good to see you are using classes and element as CSS selectors and not relying on IDs.